### PR TITLE
Add an entry for Java Round

### PR DESCRIPTION
**Pull request by Bobliuuu**
### Description

Closes issue #890. 

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by SSwiniarski** [comment]
@Dusch4593, do you know why I can't link the issue to this PR? It doesn't show up in the search.

**Review by SSwiniarski — APPROVED** [review]
@Bobliuuu, I completed a first review. This looks good and I'm passing it on to review 2.

File: content/java/concepts/math-methods/terms/round/round.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.round()`** method returns the closest integer or long to the argument.
```
Comment (HishamT) [inline content/java/concepts/math-methods/terms/round/round.md:14]: Mention that `round()` performs rounding on a number.

File: content/java/concepts/math-methods/terms/round/round.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+```java
+public class Test {
+  public static void main(String args[]) {
+    int n = Math.random(1.8);
+    System.out.println(n);
+    // Output: 2
+   }
+}
+```
```
Comment (HishamT) [inline content/java/concepts/math-methods/terms/round/round.md:36]: Use `shell` to show program output.

\```shell
2
\```

Also, try including a case when the number is rounded down.

**Comment by Bobliuuu** [comment]
Thank you for your feedback @HishamT! I made the requested changes.

File: content/java/concepts/math-methods/terms/round/round.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.round()`** method returns the closest `integer` or `long` to a number.
```
Comment (HishamT) [inline content/java/concepts/math-methods/terms/round/round.md:14]: "integer" is not a data type in Java. It is either `int` or `Integer`.

**Review by HishamT — CHANGES_REQUESTED** [review]
Looks good. Just one minor change requested.

**Comment by Bobliuuu** [comment]
Whoops, thanks, fixed! @HishamT

**Review by HishamT — APPROVED** [review]
LGTM!

**Pull request merged**